jobs/forms.py

```diff
@@ -5,10 +5,17 @@
 
 class JobForm(forms.ModelForm):
 
-    company_name = forms.CharField(max_length=150, required=True)
+    company_name = forms.CharField(label='Company',
+                                   max_length=500,
+                                   required=True)
+    website = forms.URLField(label='Company website', required=True)
 
     class Meta:
         model = Job
-        fields = ('contact_email', 'title', 'description', 'city',
-                  'country', 'expiration_date')
-        exclude = ('company',)
+        fields = ('company_name', 'website', 'contact_email', 'title',
+                  'description', 'city', 'country')
```

Comment:
I took out the expiration date - it's not something we'll want the person posting the job offer to enter.  I'm assuming it will get entered and edited only by the moderators/reviewers in the admin